tests/acceptance/stepDefinitions/loginContext.js

```diff
@@ -6,8 +6,6 @@ Given(/^the user has browsed to the login page$/,
 		const loginPage = client.page.loginPage();
 		return loginPage
 		.navigate()
-		.waitForElementVisible('@openLoginDialogButton', 1000)
```

Comment:
why are those not needed any more?

Reply:
Because, currently as soon as the user visits phoenix link, the login dialog box opens. That line waits for login button on top-right to be visible and tries to click on it.

Reply:
ok - well this will change soon anyway ... THX